buildchain/buildchain/salt_tree.py

```diff
@@ -150,6 +150,7 @@ def _run(self) -> None:
     Path('salt/metalk8s/addons/monitoring/kube-state-metrics/deployed.sls'),
     Path('salt/metalk8s/addons/monitoring/node-exporter/deployed.sls'),
     Path('salt/metalk8s/addons/monitoring/prometheus/deployed.sls'),
+    Path('salt/metalk8s/addons/monitoring/prometheus/exposed.sls'),
```

Comment:
Why separate `deployed` and `exposed` ? Currently the `exposed` state "deploy" a config map :stuck_out_tongue: 
We said few week ago that all state that should only be launched on the master should be called `deployed.sls`

Reply:
Agreed, though `deployed` is actually the upstream YAML that gets rendered through Salt into proper states... Maybe the upstream one should get renamed and `include`d in `deployed.sls`

Reply:
The idea here comes from that this state was to a large extent autogenerated from [coreos upstream](https://github.com/coreos/kube-prometheus), so I want to separate them from what's really ours.

More precisely:

1. We're not the authors per se of the contents of `deployed`, whereas we are in `exposed` or however we choose to call the ingress deployment.
2. Being "pulled" from upstream, the `deployed` files should not be modified after their initial version adjustment, whereas the `exposed` files could be, so they're not similar lifecycle-wise.

If we want to keep the naming convention on master-only states though, we can definitely have a `metalk8s.addons.monitoring.prometheus-ingress` or similar state instead - I wasn't aware of it!

Reply:
From discussion with @gdemonet, another sane option would be to store the "upstream" states in `upstream.sls` and just include these states in `deployed.sls`, where we could have our custom states for exposing services to the outside.